doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,182 @@
+
+
+.. rst-class:: numbered-step
+
+Select which ZBOSS trace logs to print
+**************************************
+
+Complete the following steps:
+
+1. Select which subsystems will receive logs by configuring the ZBOSS trace mask with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_MASK`.
+   Trace masks can be created by adding masks of subsystems to receive the trace logs from.
+   For available subsystems, see :file:`nrfxlib/zboss/production/include/zb_trace.h`.
+
+#. Select the level of logs you want to receive.
+   Configure ZBOSS trace level by selecting one of the following levels:
+
+   * Error trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_ERR`
+   * Warning trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_WRN`
+   * Info trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_INF`
+   * Debug trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_DBG`
+
+If you do not want to receive trace logs, turn them off by setting the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_OFF`.
+
+Each of the following levels on the list also includes the previous one.
+See :ref:`zigbee_ug_logging_stack_logs` to read more about trace logs.
+
+
+.. rst-class:: numbered-step
+
+Configure how to print ZBOSS trace logs
+***************************************
+
+The :ref:`zigbee_osif_zboss_osif_serial` offers a few backends to choose from for printing ZBOSS trace logs.
+It is recommended to use the Zigbee serial logger, as it is the most efficient.
+To enable it, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING`.
+
+Optional
+   You can increase size of the ring buffer that temporarily stores the trace logs.
+   To do this, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE`.
+   This can prevent losing some of the logs in demanding scenarios such as high network traffic, multiple devices being configured or joined, and so on.
+
+
+Trace logs using UART (default)
+===============================
+
+Trace logs using the UART are enabled by default.
+Below are more details on how to configure trace logs using the UART.
+
+1. Set with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_UART_LOGGING`.
+
+#. Optionally, configure which UART device you want to use with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME`.
+   The default ``UART_1`` will be used if no other UART device is configured.
+
+#. Configure the UART device that you want to use to be connected to the onboard J-Link instead of ``UART_0``, by extending the DTS overlay file for the selected board with the following:
+
+    .. code-block:: devicetree
+
+      &uart1 {
+         tx-pin = <6>;
+         rx-pin = <8>;
+         rts-pin = <5>;
+         cts-pin = <7>;
+      };
+
+      &uart0 {
+         rx-pin = <33>;
+         tx-pin = <34>;
+         /delete-property/ rts-pin;
+         /delete-property/ cts-pin;
+      };
+
+    .. note::
+      This will remove access to the application logs.
```

Comment:
```suggestion
      By connecting the UART device to the on-board J-Link, trace logs can be read directly from the J-Link COM port. As the consequence, UART device used by the logger is disconnected and application logs can not be accessed from the J-Link COM port.
```

Reply:
I have added this.
Just want to confirm that I have understood correctly that ```trace logs``` are not the same thing as ```application logs```. Is that correct?

Reply:
Yes, those two are separate logs.